wallet-core/src/main/java/eu/europa/ec/eudi/wallet/issue/openid4vci/ProcessResponse.kt

```diff
@@ -81,18 +94,55 @@ internal class ProcessResponse(
         when (outcome) {
             is SubmissionOutcome.Success -> when (val credential = outcome.credentials[0]) {
                 is IssuedCredential.Issued -> try {
-                    val cborBytes = Base64.getUrlDecoder().decode(credential.credential)
-                    logger?.d(TAG, "CBOR bytes: ${Hex.toHexString(cborBytes)}")
-                    documentManager.storeIssuedDocument(unsignedDocument, cborBytes)
-                        .notifyListener(unsignedDocument)
+                    if (isSdJwt(credential.credential)) {
+                        val certificate = parseCertificateFromSdJwt(credential.credential)
+
+                        val ecKey = ECKey.parse(certificate)
+                        val jwtSignatureVerifier = ECDSAVerifier(ecKey).asJwtVerifier()
+
+                        CoroutineScope(Dispatchers.IO).launch {
+                            SdJwtVerifier.verifyIssuance(
+                                jwtSignatureVerifier,
+                                credential.credential
+                            ).getOrThrow()
+
+                            DocumentManagerSdJwt.storeDocument(
+                                unsignedDocument.id,
+                                credential.credential
+                            )
+                            documentManager.deleteDocumentById(unsignedDocument.id)
+                            listener.invoke(
+                                IssueEvent.DocumentIssued(
+                                    unsignedDocument.id,
+                                    unsignedDocument.name,
+                                    unsignedDocument.docType
+                                )
+                            )
+                        }
+                    } else {
+                        val cborBytes = Base64.getUrlDecoder().decode(credential.credential)
+
+                        logger?.d(TAG, "CBOR bytes: ${Hex.toHexString(cborBytes)}")
```

Comment:
It's only debug level, but I think we should remove this logger here.

Reply:
Der logger ist aus der originalen lib selbst, den haben wir nicht selbst hinzugefügt, aber ich kann ihn entfernen